### PR TITLE
fix(flutter) update flutter hub documentation

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Updates Hub documentation for Flutter to reflect new API.

*KEEPING AS DRAFT UNTIL CODE IS RELEASED*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/auth/fragments/flutter/hub_events/10_listen_events.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ```dart
-auth.events.startListening((hubEvent) {
-  switch(hubEvent["eventName"]) {
+StreamSubscription hubSubscription = Amplify.Hub.listen([HubChannel.Auth], (hubEvent) {
```
Comment (Amplifiyer) [inline docs/lib/auth/fragments/flutter/hub_events/10_listen_events.md:2]: if we are not showing what to do with `StreamSubscription hubSubscription` maybe we should remove it from the docs?

Comment (haverchuck) [reply docs/lib/auth/fragments/flutter/hub_events/10_listen_events.md:2]: I felt it was useful to show what this method actually returns.  This should be recognizable and meaningful for Flutter devs.

Comment (Amplifiyer) [reply docs/lib/auth/fragments/flutter/hub_events/10_listen_events.md:2]: Can we show one usage of this variable after the call?

**Review by Amplifiyer — APPROVED** [review]

**Review by fjnoyp — APPROVED** [review]

**Review by Ashish-Nanda — APPROVED** [review]
LGTM

**Review by drochetti — APPROVED** [review]

**Pull request merged**